plugins/woocommerce/src/Internal/Admin/Settings/Payments.php

```diff
@@ -594,7 +603,113 @@ private function get_payment_gateway_base_details( WC_Payment_Gateway $payment_g
 					? sanitize_url( $payment_gateway->get_settings_url() )
 					: admin_url( 'admin.php?page=wc-settings&tab=checkout&section=' . strtolower( $payment_gateway->id ) ),
 			),
+			'onboarding'  => array(
```

Comment:
Do we need the 'onboarding` categorization?

Reply:
I agree.